pkg/porter/install.go

```diff
@@ -30,6 +30,17 @@ func (p *Porter) InstallBundle(opts InstallOptions) error {
 		return err
 	}
 
+	deperator := newDependencyExecutioner(p)
```

Comment:
`dependerator`?  `depsecutioner`? so many choices!

Reply:
I know right?! I had no clue what to call this. 😀